.github/workflows/windows_conditional_compilation.yml

```diff
@@ -105,6 +105,7 @@ jobs:
             -DENABLE_PROFILING_ITT=ON `
             -DSELECTIVE_BUILD=COLLECT `
             -DCMAKE_DISABLE_FIND_PACKAGE_PkgConfig=ON `
+            -DCMAKE_TOOLCHAIN_FILE=${{ env.OPENVINO_REPO }}/cmake/toolchains/mt.runtime.win32.toolchain.cmake `
```

Comment:
We are building samples later in this job. 

They need to use the same toolchain, otherwise we see failures now that VS runtimes mismatched. I propose setting this toolchain globally as env var for the job - in this case both samples and main build utilize this variable.